### PR TITLE
wip: M60 split

### PR DESCRIPTION
**Pull request by alinelena**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/m60_split/config.h [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+#define SPLIT_HAND_PIN B9
+#define SOFT_SERIAL_PIN B6
+#define SELECT_SOFT_SERIAL_SPEED 0
+
+#define RGB_DI_PIN B15
+
+#define RGBLED_NUM 24
+#define RGBLED_SPLIT { 12, 12 }
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/m60_split/config.h:58]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/m60_split/config.h [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+//#define ENCODER_RESOLUTION 4
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+#define MATRIX_IO_DELAY 5
+
+#define SPLIT_HAND_PIN B9
+#define SOFT_SERIAL_PIN B6
+#define SELECT_SOFT_SERIAL_SPEED 0
```
Comment (drashna) [inline keyboards/m60_split/config.h:52]: This is not used for USART based serial. 
```suggestion
#define SERIAL_USART_TX_PIN B6
```

File: keyboards/m65/keymaps/via/config.h [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#define TAPPING_TOGGLE 2
```
Comment (drashna) [inline keyboards/m65/keymaps/via/config.h:18]:
```suggestion
#pragma once

#define TAPPING_TOGGLE 2
```

File: keyboards/m65/keymaps/via/config.h [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 4
```
Comment (drashna) [inline keyboards/m65/keymaps/via/config.h:20]: This is the default and shouldn't need to be defined. 
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/m60_split/config.h [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x5A56
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    zvecr
+#define PRODUCT         split_blackpill
```
Comment (drashna) [inline keyboards/m60_split/config.h:25]: Probably want to change these. :D

**Review by drashna — CHANGES_REQUESTED** [review]
And you have some extra files n the repo, with dates attached to the extension.

File: keyboards/m60_split/config.h (hunk 5 above)
Comment (alinelena) [reply keyboards/m60_split/config.h:25]: of course once I am happy all works... i think in the end last night i managed to get it working but I need a proper testing this weekend before i send the pcbs to be printed.

**Comment by alinelena** [comment]
> And you have some extra files n the repo, with dates attached to the extension.

yes is a mess, also contains some bits from m65... i wanted to make it fast to debug what was wrong with it sunday... I will clean it up properly once I have the boards done.

**Comment by alinelena** [comment]
superseeded by https://github.com/qmk/qmk_firmware/pull/14383

**Pull request closed without merging**